Apidaze.SDK/Calls/CallState.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Apidaze.SDK.Calls
+{
+    public enum CallState
+    {
+        DOWN,
+        DIALING,
+        RINGING,
+        EARLY,
+        ACTIVE,
+        HANGUP
```

Comment:
What do we need those call states for?

Reply:
This enum was taken from Java SDK 👍

Reply:
This doesn't answer my question

Reply:
CallState is property in  Call.cs class.       [JsonProperty("callstate")] public CallState CallState { get; set; } 
I don't know exactly what is the business reasons if u ask for it as well.

Reply:
Ok, i had a look at how they are used and IMO it's not what we should do: don't know where those call states came from, but they're undocumented and could be subject to change, in which case the enum will be bogus. The SDK should return the state of the API and relay it to a consuming end user application. That application can then implement an enum if it feels brave and confident that the API won't change those states.

Reply:
So if you just return what the API gives you as an response, then you'll be safe.

Reply:
OK. I will make investigation what was returned from API and then I will fix it because there is no documentation for that?

Reply:
this is example from Python API:  'cid_name': 'Outbound Call', 'cid_num': '48504333310', 'dest': '48504333310', 'callstate': 'ACTIVE',

Reply:
Yes, so we don't know what those states are... so Python SDK returns exactly what the API response is, without mapping it to enum values

Reply:
Fine. I can change to string instead of enum type. sounds good ?